apps/roam/src/components/ExportDiscourseContext.tsx

```diff
@@ -553,6 +554,11 @@ const GraphExportButton = () => {
       icon="export"
       onClick={() => {
         const { degreesIn, degreesOut } = getGraphOverviewDepthValues();
+        posthog.capture("Discourse graph: export button clicked", {
```

Comment:
When I wrote `discourse graph exported` I didn't take into account all of the things that could mean. 😅

![image](https://github.com/user-attachments/assets/2a4a46fb-321f-4a07-a20d-e278b7f5510b)
![image](https://github.com/user-attachments/assets/34286c67-8dd3-442b-b0cd-a53fc7956b07)
![image](https://github.com/user-attachments/assets/4805cfed-9206-47bf-906b-d9f96dfae053)

This code  is explicitly for the Graph Overview Export button, so let's clarify that and table the rest of the Export options for later.

Reply:
I went ahead and renamed it, I'll leave this comment unresolved so you can see it.